sa/storage-authority.go

```diff
@@ -58,6 +58,7 @@ func NewSQLStorageAuthority(dbMap *gorp.DbMap, clk clock.Clock) (*SQLStorageAuth
 	logger := blog.GetAuditLogger()
 
 	logger.Notice("Storage Authority Starting")
+	SetSQLDebug(dbMap, logger)
```

Comment:
In prod we turn off collection of debug logs. They go to the local syslog, but not upstream. I think if we turn them on to debug a particular problem, we're likely to want to do so for specific services - and if we're debugging the SA, the queries are a valuable part of that.